v22.1/manage-certs-revoke-ocsp.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Using Online Certificate Status Protocol (OCSP) with CockroachDB
+summary: Using Online Certirifcate Status Protocol (OCSP) with CockroachDB
+toc: true
+docs_area: manage.security
+---
+
+{{ site.data.products.core }} supports [Online Certificate Status Protocol (OCSP)](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol) for certificate revocation.
+
+Read more about [Public Key Infrastructure (PKI) and Transport Layer Security (TLS) in CockroachDB](security-reference/transport-layer-security.html).
+
+To enable certificate revocation using your OCSP service:
+
+1. Ensure that your Certificate Authority sets the OCSP server address in the `authorityInfoAccess` field in the certificate.
```

Comment:
Seems like it would depend on how they are coordinating with their CA. Would be configured in the CSR if they are using them, eh? If they aren't, it's just a matter of coordination by whatever means. If that's correct, then the vagueness of "ensure..." would be pretty appropriate.
Does that sound right @catj-cockroach?

Reply:
Seems to me like this would be specified in the CSR if they are using them, or however else they are coordinating with their CA if not.
If that guess is correct, then the vagueness of "ensure..." is pretty appropriate.
@catj-cockroach whadaya think?